app/presenters/publishing_api/generic_edition_presenter.rb

```diff
@@ -29,7 +29,6 @@ def content
     def links
       LinksPresenter.new(item).extract([
         :organisations,
-        :topics,
```

Comment:
I _believe_ that this is only specialist topics. Other topics include stats announcements topics which I thought were handled differently. Otherwise a topic could also be a taxon but I _think_ in the code we usually call a taxon a taxon and only call it a topic on the user facing bit.

Let me know if I've missed any though or if I've misunderstood

Reply:
No that helps, thanks